src/utils.js

```diff
@@ -395,7 +383,7 @@ export const isAtHome = () => !!process.env[ENV_VARS.IS_AT_HOME];
  * @return {Promise}
  */
 export const sleep = (millis) => {
-    return delayPromise(millis);
+    return new Promise(res => setTimeout(res, millis));
```

Comment:
The old version in apify-shared resolved immediately if `millis` was a non-positive value, and had a unit test. You should keep the same if you really want to replace the function. See https://github.com/apifytech/apify-shared-js/blob/master/src/utilities.js#L61 and https://github.com/apifytech/apify-shared-js/blob/master/test/utilities.js#L181

Reply:
`sleep` already has a unit test that covers those cases. (0 and negative numbers).

I admit that there's a difference between `setTimeout(resolve, 0)` and `resolve`, in terms of when the promise resolves, but it's so negligible that I prefer the simpler solution.

Reply:
Alright then